src/mac.cpp

```diff
@@ -342,9 +342,31 @@ bool ActiveApplicationIsSandboxed() {
   return isSandboxed;
 }
 
+std::tuple<int, int, int, int> GetActiveApplicationWindowBounds() {
+  NSArray* windows = (NSArray*)CGWindowListCopyWindowInfo(
+    kCGWindowListOptionOnScreenOnly | kCGWindowListExcludeDesktopElements, kCGNullWindowID);
```

Comment:
should this also be `kCGWindowListOptionAll` like the other instances of this call? if so, it's probably worth creating a small utility function here, so we have a consistent way of getting the windows to iterate over.

Reply:
It should, but with an extra check to make sure the window we're getting the information for is visible and the main window. I added this and the utility function in 00c4442